rudder-web/src/main/scala/com/normation/rudder/web/model/JsInitContextLinkUtil.scala

```diff
@@ -154,12 +154,14 @@ object JsInitContextLinkUtil extends Loggable {
     nodeInfoService.getNodeInfo(id) match {
       case t: EmptyBox =>
         <span>Node {id.value}</span>
-      case Full(node) =>
```

Comment:
Since you are doing the same for emptybox and Full(None) Can you do:

case Full(some(node)) => 
case _ =>